package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+                 verbose=None):
+        """Set up calculation of water bridges between two selections in a universe.
+
+        The timeseries is accessible as the attribute :attr:`WaterBridgeAnalysis.timeseries`.
+
+        If no hydrogen bonds are detected or if the initial check fails, look
+        at the log output (enable with :func:`MDAnalysis.start_logging` and set
+        `verbose` ``=True``). It is likely that the default names for donors
+        and acceptors are not suitable (especially for non-standard
+        ligands). In this case, either change the `forcefield` or use
+        customized `donors` and/or `acceptors`.
+
+        Parameters
+        ----------
+        universe : Universe
+            Universe object
+        selection1 : str (optional)
+            Selection string for first selection ['protein']
+        selection2 : str (optional)
+            Selection string for second selection ['not resname SOL']
+            This string selects everything except water where water is assumed
+            to have a residue name as SOL.
+        water_selection : str (optional)
+            Selection string for bridging water selection ['resname SOL']
+            It is assumed that the water has a residue name SOL.
+            However, in theory this selection can be anything which forms
+            hydrogen bond with selection 1 and selection 2.
+        selection1_type : {"donor", "acceptor", "both"} (optional)
+            Selection 1 can be 'donor', 'acceptor' or 'both'. Note that the
+            value for `selection1_type` automatically determines how
+            `selection2` handles donors and acceptors: If `selection1` contains
+            'both' then `selection2` will also contain 'both'. If `selection1`
+            is set to 'donor' then `selection2` is 'acceptor' (and vice versa).
+            ['both'].
+        update_selection1 : bool (optional)
+            Update selection 1 at each frame. Setting to ``True`` if the
+            selection is not static. Selection 1 is filtered first to speed up
+            performance. Thus, setting to ``True`` is recommended if contact
+            surface between selection 1 and selection 2 is constantly
+            changing.[``False``]
+        update_selection2 : bool (optional)
+            Similiar to *update_selection1* but is acted upon selection 2.
+            [``False``]
+        update_water_selection : bool (optional)
+            Update selection for water at each frame. Setting to ``False`` is
+            **only** recommended when the total amount of water molecules in the
+            simulation are small and when water molecules remain static across
+            the simulation.
+
+            However, in normal simulations, only a tiny proportion of water is
+            engaged in the formation of water bridge. It is recommended to
+            update the water selection and set filter_first [``True``] so as to
```

Comment:
```reST
and set keyword `filter_first` to ``True``
```

Reply:
@orbeckst Thank you for the comment, I have changed the text.